README.md

```
@@ -1,93 +1,61 @@
 
-A full example can be found [here](https://github.com/ubiquibot/assistive-pricing/blob/623ea3f950f04842f2d003bda3fc7b7684e41378/tests/http/request.http).
+- `NODE_ENV=local` enables signature bypass for local development.
+- The request payload uses a Brotli+base64 `eventPayload`, matching the plugin SDK input schema.
 
-#### Deploying the Worker
+### Optional e2e (real GitHub data)
 
-For testing purposes, the worker can be deployed through the Worker Deploy and Worker Delete workflows. It requires to
-create a personal [Cloudflare Account](https://www.cloudflare.com/), and fill the `CLOUDFLARE_ACCOUNT_ID` and `CLOUDFLARE_API_TOKEN` within your
-GitHub Action Secrets.
+If you want to test against a real repository, you can use https://github.com/Meniole/daemon-task-matcher and `gh`.
```

markdownlint-cli2 (0.20.0)

[warning] Bare URL used

(MD034, no-bare-urls)